Formula/i/ircd-irc2.rb

```diff
@@ -81,3 +101,52 @@ def install
     system "#{sbin}/ircd", "-version"
```

Comment:
If possible: try add a test that uses the resolving functionality to make sure the patch doesn't break anything.

Reply:
Not sure how to test this. May just merge as is given this remaining blocker for https://github.com/Homebrew/brew/pull/18035